tests/unit/confidant/encrypted_settings_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from mock import patch
+from mock import Mock
+
+# Prevent call to KMS when app is imported
+from confidant import settings
+settings.encrypted_settings.secret_string = {}
+settings.encrypted_settings.decrypted_secrets = {'SESSION_SECRET': 'TEST_KEY'}
```

Comment:
Thinking more on this, not really. The problem is that as soon as confidant.app is imported, it will try to load the encrypted blob from the environment. I'm going to try a followup to make that not happen.